package.json

```diff
@@ -22,9 +22,10 @@
     "bootstrap": "npm install conventional-changelog-conventionalcommits",
     "changed": "lerna changed"
   },
-  "devDependencies": {
-    "@commitlint/config-conventional": "13.2.0",
+"devDependencies": {
+    "@commitlint/config-conventional": "17.6.0",
+    "conventional-changelog-conventionalcommits": "^5.0.0",
     "husky": "6.0.0",
-    "lerna": "4.0.0"
+    "lerna": "6.6.1"
```

Comment:
[clarification] Did you look through the changelog to see what the specific breaking changes are for v5 and v6? Wondering if there are any breaking changes we need to account for?